website/docs/docs/community/contributing.md

```diff
@@ -0,0 +1,77 @@
+---
+sidebar_position: 1
+---
+
+# Contributing to Kro
```

Comment:
Instead of duplicating the contributing content here (which is I think what this has done), we could symlink the CONTRIBUTING.md here.

There are some downsides to that over time, but for _now_ it's I think easy to do any avoids having to maintain duplicate docs.